appveyor.yml

```diff
@@ -1,13 +1,12 @@
-version: "{build}"
+version: '{build}'
```

Comment:
Please don't switch to single quotes in YAML files.

Reply:
:+1: